### PR TITLE
Feature/websockets

### PR DESCRIPTION
**Pull request by WouterKvG**
This PR adds websocket functionality and is capable to connect via both -api and -accounts. This should be merged after feedbackfruits/bepstore-api-goals#18 and  feedbackfruits/bepstore-api#37.

### CODE REVIEW

**Comment by WouterKvG** [comment]
Please check @steffansluis and @bartist

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@
 lcov.info
 npm-debug.log
 testem.log
+dump.rdb
```
Comment (steffansluis) [inline .gitignore]: Where did this come from?

File: app/adapters/activity.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import ApplicationAdapter from './application';
```
Comment (steffansluis) [inline app/adapters/activity.js]: Doesn't feedbackfruits-ui-activities already provide an Activity model that has the endpoints `/v1/activities`? Can't we use that?

File: package.json [hunk 3]
```diff
@@ -65,6 +66,7 @@
     "ember-resolver": "^2.0.3",
     "ember-simple-auth": "1.1.0",
     "ember-truth-helpers": "1.2.0",
+    "ember-websockets": "3.0.0",
```
Comment (steffansluis) [inline package.json]: This wasn't needed anymore right?

File: tests/integration/components/simple-chat-test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+/* jshint expr:true */
```
Comment (steffansluis) [inline tests/integration/components/simple-chat-test.js]: Do we have a simple-chat component?

File: .gitignore (hunk 1 above)
Comment (WouterKvG) [inline .gitignore]: redis

File: app/adapters/activity.js (hunk 2 above)
Comment (WouterKvG) [inline app/adapters/activity.js]: Probably yes :)

**Pull request merged**